app/page/index.js

```diff
@@ -2,20 +2,81 @@ import { BasePage } from '@zeppos/zml/base-page';
 import * as appService from '@zos/app-service';
 import { queryPermission, requestPermission } from '@zos/app';
 import hmUI from '@zos/ui';
-import { START_BUTTON } from 'zosLoader:./index.[pf].layout.js';
-import { STOP_BUTTON } from './index.r.layout';
+import { push } from '@zos/router';
+import { Sleep } from '@zos/sensor'; // Import the Sleep module
+import {
+	START_BUTTON,
+	SLEEP_BUTTON,
+	PERMISSIONS_BUTTON,
+	STOP_BUTTON,
+} from 'zosLoader:./index.[pf].layout.js';
 
 const permissions = ['device:os.bg_service'];
 const service = 'app-service/service';
-const storage = getApp().globals.storage;
+const storage = getApp()._options.globalData.storage;
```

Comment:
think a merge conflict changed this, it should be `const storage = getApp().globals.storage`